source/Government.cpp

```diff
@@ -180,6 +176,8 @@ void Government::Load(const DataNode &node)
 			language = child.Token(valueIndex);
 		else if(key == "raid")
 			raidFleet = GameData::Fleets().Get(child.Token(valueIndex));
+		else if(key == "enforces" && child.Token(valueIndex) == "all")
+			child.PrintTrace("Error: Skipping unsupported \"enforces all\" syntax:");
```

Comment:
This should still work, just warn that it's deprecated.
```suggestion
		else if(key == "enforces" && child.Token(valueIndex) == "all")
		{
			enforcementZones.clear();
			child.PrintTrace("Warning: Deprecated use of \"enforces all\" instead of \"remove enforces\":");
		}
```